app/models/team_zip_export.rb

```diff
@@ -30,7 +30,10 @@ def generate_exportable_zip(user, data, type, options = {})
     fill_content(dir_to_zip, data, type, options)
     zip!(dir_to_zip, output_file.path)
     self.zip_file = File.open(output_file)
-    generate_notification(user) if save
+    if save
+      FileUtils.rm_rf([dir_to_zip, output_file], secure: true)
```

Comment:
Maybe it makes sense to put it inside 'ensure' block to remove files even in case of error?